tests/generic_config_updater/test_lo_interface.py

```diff
@@ -19,15 +20,36 @@
 # Loopback0                         fc00:1::32/128                              up/up         N/A             N/A
 #                                   fe80::4a3:18ff:fec2:f9e3%Loopback0/64                     N/A             N/A
 
+DEFAULT_LOOPBACK = "Loopback0"
+REPLACE_IP = "10.1.0.210/32"
+REPLACE_IPV6 = "FC00:1::210/128"
+
 pytestmark = [
     pytest.mark.topology('t0'),
 ]
 
 logger = logging.getLogger(__name__)
 
 
+@pytest.fixture(scope="module")
+def lo_intf(cfg_facts):
+    def _is_ipv4_address(ip_addr):
+        return ipaddress.ip_address(ip_addr).version == 4
```

Comment:
>ip_address [](http://example.com/codeflow?start=25&length=10)

you can directly use `ip_network`. No need to split.

Reply:
Tried with the local test. Seems ip_network needs to UNSET the host bit.
For example, "10.1.0.32/31" is good to use while "10.1.0.33/31" is not.
```
ip = "10.1.0.32/31"
print(type(ip_network(ip)) is IPv4Network) # True

ip = "10.1.0.33/31"
print(type(ip_network(ip)) is IPv4Network) # raise ValueError('%s has host bits set' % self)
                                             ValueError: 10.1.0.33/31 has host bits set
```
There is no problem with changing to ip_network here. Because all loopback has a mask of "/32". 
I suggest keeping the `split` part to handle all mask and host issues.